release notes/v0.27.0.md

```diff
@@ -0,0 +1,165 @@
+            maxVUs: 100, // if the preAllocatedVUs are not enough, we can initialize more
+
+            tags: { test_type: 'api' }, // different extra metric tags for this scenario
+            env: { MY_CROC_ID: '2' }, // same function, different environment variables
+            exec: 'apitest', // same function as the scenario above, but with different env vars
+        },
+    },
+    discardResponseBodies: true,
+    thresholds: {
+        // we can set different thresholds for the different scenarios because
+        // of the extra metric tags we set!
+        'http_req_duration{test_type:api}': ['p(95)<250', 'p(99)<350'],
+        'http_req_duration{test_type:website}': ['p(99)<500'],
+        // we can reference the scenario names as well
+        'http_req_duration{scenario:my_api_test_2}': ['p(99)<300'],
+    }
+};
+
+export function webtest() {
+    http.get('https://test.k6.io/contacts.php');
+    sleep(Math.random() * 2);
+}
+
+export function apitest() {
+    http.get(`https://test-api.k6.io/public/crocodiles/${__ENV.MY_CROC_ID}/`);
+    // no need for sleep() here, the iteration pacing will be controlled by the
+    // arrival-rate executors above!
+}
+```
+
+As shown in the example above and the [documentation](https://k6.io/docs/using-k6/scenarios), all executors have some additional options that improve their flexibility and facilitate code reuse, especially in multi-scenario test runs:
+- Each executor has a `startTime` property, which defines at what time, relative to the beginning of the whole test run, the scenario will start being executed. 
+- Executors have a new `gracefulStop` property that allows for iterations to complete gracefully for some amount of time after the normal executor duration is over ([#879](https://github.com/loadimpact/k6/issues/879), [#1033](https://github.com/loadimpact/k6/issues/1033)). The `ramping-vus` executor additionally also has `gracefulRampDown`, to give iterations time to finish when VUs are ramped down. The default value for both options is `30s`, so it's a slight breaking change, but the old behavior of immediately interrupting iterations can easily be restored by setting these options to `0s`.
+- Different executors can execute different functions other than the `default` exported one. This can be specified by the `exec` option in each `scenarios` config, and allows for more flexibility in organizing your tests, easier code reuse, building test suites, etc.
+- To allow for even greater script flexibility and code reuse, you can specify different environment variables and tags in each scenario, via the new `env` and `tags` executor options respectively.
+- k6 will now emit a new `dropped_iterations` metric in `shared-iterations`, `per-vu-iterations`, `constant-arrival-rate` and `ramping-arrival-rate` executors if it can't schedule an iteration on time, depending on the configured rates and the iteration duration ([#1529](https://github.com/loadimpact/k6/pull/1529)).
```

Comment:
```suggestion
- k6 may now emit a new `dropped_iterations` metric in the `shared-iterations`, `per-vu-iterations`, `constant-arrival-rate` and `ramping-arrival-rate` executors; this is done if it can't run an iteration on time, depending on the configured rates (for the arrival-rate executors) or scenario `maxDuration` (for the iteration-based executors), so it's generally a sign of a poor config or an overloaded system under test ([#1529](https://github.com/loadimpact/k6/pull/1529)).
```